src/teams.rs

```diff
@@ -4,84 +4,232 @@ use std::collections::BTreeMap;
 
 use diesel::prelude::*;
 use toml;
+use serde::de;
 
 use super::DB_POOL;
 use domain::github::GitHubUser;
 use error::*;
 
+//==============================================================================
+// Public API
+//==============================================================================
+
 lazy_static! {
-    pub static ref TEAMS: Teams = {
-        let teams_file =
-            include_str!(concat!(env!("CARGO_MANIFEST_DIR"), "/teams.toml"));
-        let teams_from_file: TeamsFromFile =
-            toml::from_str(teams_file).expect("couldn't parse teams");
-
-        let mut teams = BTreeMap::new();
-
-        for (label, team_from_file) in teams_from_file {
-            let label = TeamLabel(label);
-            let team = team_from_file.validate()
-                .expect("unable to verify team member from database.
-if you're running this for tests, make sure you've pulled github users from prod");
-            teams.insert(label, team);
-        }
+    pub static ref SETUP: RfcbotConfig = read_rfcbot_cfg_validated();
+}
+
+#[derive(Debug, Deserialize)]
+pub struct RfcbotConfig {
+    fcp_behaviors: BTreeMap<String, FcpBehavior>,
+    teams: BTreeMap<TeamLabel, Team>,
+}
+
+impl RfcbotConfig {
+    /// Retrive an iterator over all the team labels.
+    pub fn team_labels(&self) -> impl Iterator<Item = &TeamLabel> {
+        self.teams.keys()
+    }
+
+    /// Retrive an iterator over all the (team label, team) pairs.
+    pub fn teams(&self) -> impl Iterator<Item = (&TeamLabel, &Team)> {
+        self.teams.iter()
+    }
+
+    /// Are we allowed to auto-close issues after F-FCP in this repo?
+    pub fn ffcp_auto_close(&self, repo: &str) -> bool {
```

Comment:
It's not clear to me from the function name that it's answering a question. Maybe something like `should_finished_fcp_auto_close`? I'm sure that without the abbreviation lots of things will be too long, but in this case the extra characters would make it much more readable.

Reply:
I went with the compromise `should_ffcp_auto_close`; 5 words is a bit too long for me ^,-